pyaedt/modeler/cad/component_array.py

```diff
@@ -0,0 +1,843 @@
+            return self._cells
+
+        if self._app.settings.aedt_version > "2023.2":  # pragma: no cover
+            self.export_array_info(array_path=None)
+            self._array_info_path = os.path.join(self._app.toolkit_directory, "array_info.csv")
+
+        self._cells = [[None for _ in range(self.b_size)] for _ in range(self.a_size)]
+        array_props = self._array_props
+        component_names = self.component_names
+        for row_cell in range(0, self.a_size):
+            for col_cell in range(0, self.b_size):
+                self._cells[row_cell][col_cell] = CellArray(row_cell, col_cell, array_props, component_names, self)
+        return self._cells
+
+    @property
+    def name(self):
+        """Name of the array.
+
+        Returns
+        -------
+        str
+           Name of the array.
+        """
+        return self._m_name
+
+    @name.setter
+    def name(self, array_name):
+        if array_name not in self._app.component_array_names:
+            if array_name != self._m_name:
+                self._oarray.SetPropValue("Name", array_name)
+                self._app.component_array.update({array_name: self})
+                self._app.component_array_names = list(self._app.omodelsetup.GetArrayNames())
+                del self._app.component_array[self._m_name]
+                self._m_name = array_name
+
+        else:  # pragma: no cover
+            self._logger.warning("Name %s already assigned in the design", array_name)
+
+    @property
+    def post_processing_cells(self):
+        """Postprocessing cells.
+
+        Returns
+        -------
+        dict
+           Postprocessing cells of each component.
+        """
+        if not self._post_processing_cells:
+            self._post_processing_cells = {}
+            component_info = {}
+            row = 1
+            for row_info in self.cells[:]:
+                col = 1
+                for col_info in row_info:
+                    name = col_info.component
+                    if name not in component_info:
+                        component_info[name] = [[row, col]]
+                    else:
+                        component_info[name].append([row, col])
+                    col += 1
+                row += 1
+
+            for component_name, component_cells in component_info.items():
+                if component_name not in self._post_processing_cells.keys() and component_name is not None:
+                    self._post_processing_cells[component_name] = component_cells[0]
+
+        return self._post_processing_cells
+
+    @post_processing_cells.setter
+    def post_processing_cells(self, val):
+        if isinstance(val, dict):
+            self._post_processing_cells = val
+            self._edit_array()
+
+        else:  # pragma: no cover
+            self._logger.error("Dictionary with component names and cell not correct")
+
+    @property
+    def visible(self):
+        """Array visibility.
```

Comment:
```suggestion
        """Flag indicating if the array is visible.
```
Properties should be described as nouns.